File: backend/Multibanca.Domain.Models/Multibanca/registrar_firma_apoderado_banco.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca
{
    public class registrar_firma_apoderado_banco : base_auditoria
    {
        public int id_registrar_firma_apoderado_banco { get; set; }
        public DateTime fecha_firma { get; set; }
        public long id_expediente { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/registrar_firma_apoderado_banco/models/registrar_firma_apoderado_banco.ts
export interface RegistrarFirmaApoderadoBanco {
  id_registrar_firma_apoderado_banco: number;
  fecha_firma: string;  // DateTime en C# se convierte a string en TypeScript
  id_expediente: number;
  observaciones?: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}